mattumland/memento
# Testing

## Is this a fix or a feature?
Feature
## What is the change?
Adds cypress test suite
## What does it fix?
n/a
## Where should the reviewer start?
In the cypress directory -> integration -> memento.spec.js
## How should this be tested?
- Open cypress in terminal (yarn/npn run cypress open)
- Select memento.spec.js
- Watch for green check marks
